getting-started.md

```diff
@@ -1,35 +1,45 @@
 # Getting Started
 
-## Creating Account
+### Creating Account
 
-You will need the Exogress account to use the service. Please, visit [https://app.exogress.com](https://app.exogress.com/) and create a new account.
+Visit [https://app.exogress.com](https://app.exogress.com/) to create a new account and follow the instructions. If you signed up using email, make sure you have activated your account.
 
-## Creating Project
+### Choose your account and subdomain names
 
-Project is the place where your configurations live in. You will be prompted to create your first project once your account is ready. Later, you'll be able to add more projects.
+You will be prompted to provide your account name. This will be associated with your third-level domain name in Exogress, and couldn't be changed.
+Next, we ask you to specify a fourth-level subdomain - this will be the first default url. These domains are free and you could always add your own domain at any time later.
```

Comment:
sounds like it's a part of the registration flow. in fact, you can create multiple 4th level subdomains in "Domains" menu section at any time.